src/graph.rs

```diff
@@ -5503,7 +5503,7 @@ impl<'a, 'graph> Builder<'a, 'graph> {
                 count: redirect_count + 1,
                 specifier,
                 maybe_attribute_type,
-                is_asset: false,
+                is_asset,
```

Comment:
I extracted out this code from regular loading to be shared with the asset loading code and missed replacing this.